### ISSUE TITLE
mifi_set_default_loglevel has no effect

### ISSUE THREAD

**Issue by heikoklein** [metno/fimex#39]
Calling `mifi_set_default_loglevel` has no effect when used from the fortran interface.

`mifi_set_default_loglevel` in c_fimex.cc sets a variable which, when having log4cpp, is only used on the first initialization and not when the log-level is changed. I guess that adding `Logger::setClass(Logger::LOG4CPP);` should be called after setting defaultLogLevel, i.e.
``` c++
void mifi_set_default_log_level(int loglevel)
{
    defaultLogLevel(static_cast<Logger::LogLevel>(loglevel));
    Logger::setClass(Logger::LOG4CPP);
}
```

**Comment by alexander-buerger-met-no** [comment]
Does it help to call the function earlier in the program? It seems to work for `fiXYcontents`.

**Comment by heikoklein** [comment]
It is the first statement (after variable declaration) in the fortran code at https://github.com/metno/snap/blob/master/src/common/snap.F90#L289 . It is not possible to call it any earlier in Fortran.

**Comment by alexander-buerger-met-no** [comment]
This should be solved with commit f0cf6736.
